templates/hardwarelab/partials/manager_partial.html

```diff
@@ -0,0 +1,79 @@
+<div class="ui segment">
```

Comment:
This partial is no longer needed but left in case we need to revert to using `.replaceWith()` for updating the views